src/components/modals/CopyToClipboard.vue

```diff
@@ -0,0 +1,83 @@
+<template lang="html">
+
+  <b-modal hide-footer visible v-bind:title="$t('modal_title_share_article')">
+
+    <iframe :src="widgetLink"
+      class="widgetLink border">
+      <p>Your browser does not support iframes.</p>
+    </iframe>
+
+    <b-input-group prepend="URL">
+      <b-form-input id="inputLink" ref="inputLink" readonly v-model="widgetLink" />
+      <template v-slot:append>
+        <b-button
+          variant="outline-primary" size="sm"
+          v-on:click="copyArticleUrlToClipboard()"
+          >{{ $t('copy_to_clipboard') }}
+        </b-button>
+      </template>
+    </b-input-group>
+
+  </b-modal>
+
+</template>
+
+<script>
+export default {
+  model: {
+    prop: 'article',
+  },
+  props: ['article'],
+  computed: {
+    widgetLink() {
+      const baseURL = (location.hostname === 'localhost' || location.hostname === '127.0.0.1') ?
```

Comment:
`baseUrl` should use `this.publicPath` (see https://cli.vuejs.org/config/#publicpath). So you don't need to use a ternary operator here, just `${window.location.origin}/${this.publicPath}`

Reply:
added publicPath variable from `config`